worker/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2021",
     "lib": ["es2021"],
     "module": "es2022",
-    "moduleResolution": "nodenext",
+    "moduleResolution": "node",
```

Comment:
why not nodenext?

Reply:
It should really be bundler, but Wrangler doesn't support TS5. nodenext requires file extensions.